src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1247,4 +1247,10 @@ This version used in: {2}</value>
   <data name="default_expression_can_be_simplified" xml:space="preserve">
     <value>'default' expression can be simplified</value>
   </data>
+  <data name="Format_string_contains_invalid_placeholder_0" xml:space="preserve">
+    <value>Format string contains invalid placeholder '{0}'</value>
```

Comment:
Nit: does hte {0} add anything meaningful to the user here?  They'll already get the right section squiggled, right?  So the message doesn't need this info.  Up to you.